helm/docs/expose-teams-api.md

```diff
@@ -12,18 +12,18 @@
 
 ---
 
-# Exposing the `teams-api` Service
+# Exposing the `enterprise-api` Service
```

Comment:
this will still be called the `teams-api` service

Reply:
Thanks. Just a bad find-replace

Reply:
yep, figured (you were pretty careful not to do it anywhere else)